### PR TITLE
Remove IS References

### PR DESCRIPTION
**Pull request by DonOmalVindula**
## Purpose
$subject

## Related Issue
- https://github.com/wso2-enterprise/asgardeo-product/issues/413

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,51 +28,42 @@ You can experience the capabilities of Asgardeo Tomcat OIDC Agent by following t
  sections listed below.
 
   * [Prerequisites](#prerequisites)
-  * [Configuring Identity Server](#configuring-identity-server)
-  * [Configuring the sample](#configuring-the-sample)
-  * [Running the sample](#running-the-sample)
+  * [Create Application in Asgardeo](#1-create-application-in-asgardeo)
```
Comment (brionmario) [inline README.md]:
```suggestion
  * [Create Application in Asgardeo](#create-an-application-in-asgardeo)
```

File: README.md [hunk 2]
```diff
@@ -28,51 +28,42 @@ You can experience the capabilities of Asgardeo Tomcat OIDC Agent by following t
...
+  * [Running the sample apps](#2-running-the-sample-apps)
```
Comment (brionmario) [inline README.md:32]: Do we need the `#2`? 🤔
If it works, then we can keep it?

File: README.md [hunk 3]
```diff
@@ -28,51 +28,42 @@ You can experience the capabilities of Asgardeo Tomcat OIDC Agent by following t
...
-1. Start the WSO2 IS. 
-2. Access WSO2 IS management console and create a service provider (ex:- oidc-sample-app)
-   
-   For the service provider, configure Oauth/OpenID Connect under Inbound Authentication Configuration. In this
-    configuration,
-   use following parameters and options,
-     
-       Callback URL - regexp=(http://localhost:8080/oidc-sample-app/oauth2client|http://localhost:8080/oidc-sample-app/index.html)
-
+### 1. Create Application in Asgardeo
```
Comment (brionmario) [inline README.md]:
```suggestion
### 1. Create an Application in Asgardeo
```

Comment (DonOmalVindula) [reply README.md]: Fixed with 5344aa6f90381231efb520b30bef3716331624bd

File: README.md (hunk 1 above)
Comment (DonOmalVindula) [reply README.md]: Fixed with 5344aa6f90381231efb520b30bef3716331624bd

File: README.md (hunk 2 above)
Comment (DonOmalVindula) [reply README.md:32]: Yes. Having the numbers does work. Therefore decided to keep the `#1` and `#2` to match the topics.

**Review by brionmario — APPROVED** [review]

**Review by brionmario — APPROVED** [review]

**Pull request merged**